Plugin: 群聊语录库

**插件名称：**

群聊语录库

**插件功能：**

一款QQ群语录库——支持上传聊天截图为语录，随机投放语录，关键词搜索语录精准投放

**PyPI 项目名：**

nonebot-plugin-quote

**插件 import 包名：**

nonebot_plugin_quote

**插件项目仓库/主页链接：**

https://github.com/RongRongJi/nonebot_plugin_quote

**标签：**

[{"label":"a:onebot","color":"#326bad"},{"label":"t:语录","color":"#003f88"}]

# 📃 商店发布检查结果

> Plugin: 群聊语录库

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-quote 的信息如下：
    name         : nonebot-plugin-quote 
     version      : 0.1.4                
     description  : 一款适用于QQ群聊天的语录库插件     
    
    dependencies
     - jieba &gt;=0.42.1,&lt;0.43.0
     - nonebot2 &gt;=2.0.0rc3,&lt;3.0.0
插件 nonebot-plugin-quote 依赖的插件如下：
    
插件 nonebot_plugin_quote 加载出错：
    03-17 15:55:22 [SUCCESS] nonebot | NoneBot is initializing...
    03-17 15:55:22 [INFO] nonebot | Current Env: prod
    03-17 15:55:22 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_quote&quot;
    Traceback (most recent call last):
      File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
        valid = load_plugin(&quot;nonebot_plugin_quote&quot;)
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
        return manager.load_plugin(module_path)
    &gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
        module = importlib.import_module(name)
      File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
        super().exec_module(module)
      File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_quote/__init__.py&quot;, line 6, in &lt;module&gt;
        from nonebot.adapters.onebot.v11 import Bot, Event, Message, MessageEvent, PrivateMessageEvent, MessageSegment, GroupMessageEvent
    ModuleNotFoundError: No module named &#x27;nonebot.adapters.onebot&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: a:cqhttp-#326bad, t:语录-#003f88。</li><li>✅ 项目 <a href="https://github.com/RongRongJi/nonebot_plugin_quote">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-quote/">nonebot-plugin-quote</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

> # 📃 商店发布检查结果
> > Plugin: 群聊语录库
> 
> **⚠️ 在发布检查过程中，我们发现以下问题：**
> 
> ```
> * ⚠️ 插件加载测试未通过。测试输出插件 nonebot-plugin-quote 的信息如下：
>       name         : nonebot-plugin-quote 
>        version      : 0.1.5                
>        description  : 一款适用于QQ群聊天的语录库插件     
>       
>       dependencies
>        - jieba >=0.42.1,<0.43.0
>        - nonebot-adapter-onebot >=2.2.1,<3.0.0
>        - nonebot2 >=2.0.0rc3,<3.0.0
>   插件 nonebot-plugin-quote 依赖的插件如下：
>       
>   插件 nonebot_plugin_quote 加载出错：
>       03-17 16:28:20 [SUCCESS] nonebot | NoneBot is initializing...
>       03-17 16:28:20 [INFO] nonebot | Current Env: prod
>       03-17 16:28:20 [ERROR] nonebot | Failed to import "nonebot_plugin_quote"
>       Traceback (most recent call last):
>         File "/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py", line 4, in <module>
>           valid = load_plugin("nonebot_plugin_quote")
>         File "/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py", line 34, in load_plugin
>           return manager.load_plugin(module_path)
>       > File "/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py", line 141, in load_plugin
>           module = importlib.import_module(name)
>         File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
>           return _bootstrap._gcd_import(name[level:], package, level)
>         File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
>         File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
>         File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
>         File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
>         File "/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py", line 233, in exec_module
>           super().exec_module(module)
>         File "<frozen importlib._bootstrap_external>", line 883, in exec_module
>         File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
>         File "/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_quote/__init__.py", line 12, in <module>
>           import requests
>       ModuleNotFoundError: No module named 'requests'
> ```
> 
> 详情
> 💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。 💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。
> 
> ♻️ 评论已更新至最新检查结果
> 
> 💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)

已追加onebot

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4839,5 +4839,24 @@
     "homepage": "https://github.com/NanakoOfficial/nonebot_plugin_randomnana",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_quote",
+    "project_link": "nonebot-plugin-quote",
+    "name": "群聊语录库",
+    "desc": "一款QQ群语录库——支持上传聊天截图为语录，随机投放语录，关键词搜索语录精准投放",
+    "author": "RongRongJi",
+    "homepage": "https://github.com/RongRongJi/nonebot_plugin_quote",
+    "tags": [
+      {
+        "label": "a:onebot",
+        "color": "#326bad"
+      },
+      {
+        "label": "t:语录",
+        "color": "#003f88"
+      }
+    ],
+    "is_official": false
   }
 ]

